### PR TITLE
[QC-469] Refactor repocleaner to have a proper python module

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

File: Framework/script/RepoCleaner/qcrepocleaner/Ccdb.py [hunk 1]
```diff
@@ -57,15 +57,15 @@ def __init__(self, url):
...
+    def getObjectsList(self, added_since: int = 0, path: str = "") -> List[str]:
         '''
         Get the full list of objects in the CCDB that have been created since added_since.
 
         :param added_since: if specified, only return objects added since this timestamp in epoch milliseconds.
         :return A list of strings, each containing a path to an object in the CCDB.
         '''
         logger.debug(f"added_since : {added_since}")
-        url_for_all_obj = self.url + '/latest/.*'
+        url_for_all_obj = self.url + '/latest/' + path + '/.*'
```
Comment (knopers8) [inline Framework/script/RepoCleaner/qcrepocleaner/Ccdb.py:68]: if the path is empty, we get double "/", is this fine?

File: Framework/script/RepoCleaner/qcrepocleaner/config.yaml [hunk 2]
```diff
@@ -25,9 +25,9 @@ Rules:
 #  - object_path: Test
 #    delay: 240
 #    policy: none_kept
-#  - object_path: .*
-#    delay: 1440
-#    policy: skip
+  - object_path: qc/DAQ/MO/daqTask/numberRDHs
+    delay: 1440
+    policy: none_kept
```
Comment (knopers8) [inline Framework/script/RepoCleaner/qcrepocleaner/config.yaml:30]: did you enable this only for a test or this is actually needed?

**Review by knopers8 — COMMENTED** [review]
I do not know much about proper python packaging, but it looks sane.

File: Framework/script/RepoCleaner/qcrepocleaner/Ccdb.py (hunk 1 above)
Comment (Barthelemy) [reply Framework/script/RepoCleaner/qcrepocleaner/Ccdb.py:68]: yes, I tested this case

File: Framework/script/RepoCleaner/qcrepocleaner/config.yaml (hunk 2 above)
Comment (Barthelemy) [reply Framework/script/RepoCleaner/qcrepocleaner/config.yaml:30]: good catch, that was just for a test, I will remove it

**Review by knopers8 — APPROVED** [review]

**Pull request merged**